docs/_documentations/codewind-minikube-guickguide.md

```diff
@@ -0,0 +1,181 @@
+    * **Note:** Make sure Docker is running. Otherwise, you get an error.
+4. Choose the `Open Liberty (Default templates)`.
+5. Name your project `appsody-calculator`.
+    * If you don’t see Appsody templates, find and select `Template Source Manager` and enable `Appsody Stacks - appsodyhub`. The templates are refreshed, and the Appsody templates are available.
+6. Press Enter.
+    * To monitor your project’s progress, right-click your project, and select `Show all logs`. Then an `Output` tab is displayed where you see your project’s build logs.
+
+Your project is complete when you see your application status is running and your build status is successful.
+
+### Accessing the application endpoint in a browser
+
+1. Return to your project under the Explorer pane.
+2. Select the Open App icon next to your project's name, or right-click your project and select `Open App`.
+
+Your application is now opened in the browser, showing the welcome to your Appsody microservice page.
+
+### Adding a REST service to your application
+
+ 1. Go to your project's workspace under the Explorer tab.
+ 2. Navigate to `src->main->java->dev->appsody->starter`.
+ 3. Right-click `starter` and select `New File`.
+ 4. Create a file, name it `Calculator.java`, and press `Enter`. This file is your JAX-RS resource.
+ 5. Populate the file with the following code then **save** the file:
+
+```
+package dev.appsody.starter;
+
+import javax.ws.rs.core.Application;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import javax.ws.rs.PathParam;
+
+@Path("/calculator")
+public class Calculator extends Application {
+
+    @GET
+    @Path("/aboutme")
+    @Produces(MediaType.TEXT_PLAIN)
+    public String aboutme(){
+        return "You can add (+), subtract (-), and multiply (*) with this simple calculator.";
+    }
+
+    @GET
+    @Path("/{op}/{a}/{b}")
+    @Produces(MediaType.TEXT_PLAIN)
+    public Response calculate(@PathParam("op") String op, @PathParam("a") String a, @PathParam("b") String b)
+    {
+        int numA = Integer.parseInt(a);
+        int numB = Integer.parseInt(b);
+
+      switch(op)
+      {
+          case "+":
+              return Response.ok(a + "+" + b + "=" + (Integer.toString((numA + numB)))).build();
+
+          case "-":
+              return Response.ok(a + "-" + b + "=" + (Integer.toString((numA - numB)))).build();
+
+          case "*":
+              return Response.ok(a + "*" + b + "=" + (Integer.toString((numA * numB)))).build();
+
+          default:
+              return Response.ok("Invalid operation. Please Try again").build();
+      }
+    }
+}
+```
+Any changes you make to your code is automatically built and re-deployed by Codewind and viewable in your browser.
+
+### Working with the microservice
+
+You now can work with your calculator.
+
+1. Use the port number you saw when you first opened the application.
```

Comment:
Change this to something like "Use the URL you saw when you first opened the application"